pkg/listwatch/namespace_denylist_test.go

```diff
@@ -92,8 +160,7 @@ func TestDenylistList(t *testing.T) {
 					t.Error(err)
 					return
 				}
-
-				got = append(got, acc.GetNamespace())
+				got = append(got, getNamespace(acc))
```

Comment:
Bit below https://github.com/coreos/prometheus-operator/blob/28db7c57b645a9f4827e5922a558765ac6b33a3c/pkg/listwatch/namespace_denylist_test.go#L179 we are using the `acc.GetNamespace` should we use the `getNamespace()` there as well?